Tools/scripts/run_astyle.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+"""
+Runs astyle over directory sub-trees known to be "astyle-clean"
+
+ AP_FLAKE8_CLEAN
+"""
+
+import os
+import subprocess
+import sys
+
+import argparse
+
+os.environ['PYTHONUNBUFFERED'] = '1'
+
+
+class AStyleChecker(object):
+    def __init__(self):
+        self.retcode = 0
+        self.directories_to_check = [
+            'libraries/AP_DDS',
+        ]
+        self.files_to_check = []
+
+    def progress(self, string):
+        print("****** %s" % (string,))
+
+    def check(self):
+        '''run astyle on all files in self.files_to_check'''
+        for path in self.files_to_check:
+            self.progress("Checking (%s)" % path)
```

Comment:
Now that we have f-strings, want to use them?

Reply:
Next time :-)